app/scripts/controllers/permissions/background-api.js

```diff
@@ -6,8 +6,8 @@ import {
 import {
   Caip25CaveatType,
   Caip25EndowmentPermissionName,
-  setPermittedChainIds,
-  setPermittedAccounts,
+  setPermittedChainIds as setPermittedChainIdsOnCaveatValue,
+  setPermittedAccounts as setPermittedAccountsOnCaveatValue,
```

Comment:
These methods will be renamed to `setChainIdsInCaip25CaveatValue` and `setNonSCACaipAccountIdsInCaip25CaveatValue` in the new version of the package... Should we merge that update first?